### DISCUSSION TITLE
Does Net.serve use http or https?

### DISCUSSION THREAD

**Discussion by Kyllingene** [arturo-lang/arturo#387] [Q&A] [answered]
The question is pretty much the title; does `serve` use http or https? I assume http, and in that case is there any potential way to serve an https webpage through arturo? (not critical, this is obviously not professional hosting software, at least not yet)

**Comment by drkameleon** [comment, marked as answer]
It uses HTTP.

However, as a sidenote, let me say that the Net module (and practically, anything related to the Web or Web app - to be precise - side of Arturo) is exactly what I'm going to work on as a priority in the next few weeks (or as soon as I find some real free time). I see a lot of potential in that; plus, as I've already said, it's one of the main reasons why Arturo came into existence in the first place: to be aple to create portable, no-dependencies, webview-type GUI apps easily and effeciently. 😉

**Reply by Kyllingene** [reply]
Thanks! I agree, Arturo really could work wonders in that department; it really is quite easy to use.
